ubiquibot-config-default.json

```diff
@@ -51,7 +51,38 @@
       "weight": 5
     }
   ],
+  "command-settings": [
+    {
+      "name": "start",
+      "enabled": false
+    },
+    {
+      "name": "stop",
+      "enabled": false
+    },
+    {
+      "name": "wallet",
+      "enabled": false
+    },
+    {
+      "name": "payout",
+      "enabled": false
+    },
+    {
+      "name": "multiplier",
+      "enabled": false
+    },
+    {
+      "name": "query",
+      "enabled": false
+    },
+    {
+      "name": "allow",
+      "enabled": false
+    }
+  ],
   "default-labels": ["Time: <1 Hour", "Priority: 0 (Normal)", "Test"],
+  "assistive-pricing": false,
   "auto-pay-mode": true,
   "disable-analytics": true,
```

Comment:
```suggestion
```

This should be the only hidden property so that we don't remind partners to opt-out, basically.